tests/xharness/Jenkins.cs

```diff
@@ -180,6 +181,13 @@ IEnumerable<TestData> GetTestData (RunTestTask test)
 				case "monotouch-test":
 					yield return new TestData { Variation = "Release (all optimizations)", MTouchExtraArgs = "--registrar:static --optimize:all", Debug = false, Profiling = false, Defines = "OPTIMIZEALL" };
 					yield return new TestData { Variation = "Debug (all optimizations)", MTouchExtraArgs = "--registrar:static --optimize:all", Debug = true, Profiling = false, Defines = "OPTIMIZEALL" };
+					yield return new TestData { Variation = "Debug (interpreter)", MTouchExtraArgs = "--interpreter", Debug = true, Profiling = false, Ignored = true, };
```

Comment:
@rolfbjarne why isn't `Undefines = "FULL_AOT_RUNTIME" ` needed here?

Reply:
@lewurm the monotouch-test tests don't use/define `FULL_AOT_RUNTIME`, so it would be unnecessary to try to remove it.

In any case removing the `FULL_AOT_RUNTIME` is not needed, I just guessed that it would end up include tests that might work fine with the interpreter.